pkgs/tools/misc/bashplotlib/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, pythonPackages, fetchFromGitHub }:
+
+pythonPackages.buildPythonApplication rec {
+  name = "bashplotlib-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed. Just add pname and version